Bottom: e7cdbb46f7a62074dcc257e7ebd058449de044ea
Top:    9c3fea57ef532e7164126bc1d70edc8f8253ed8c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 00:40:24 +0000

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index fa03ba7..9f445f1 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2611,7 +2611,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
